app/src/main/res/values/strings.xml

```diff
@@ -1183,7 +1183,7 @@
     <string name="rt_cancel_btn">Cancel</string>
     <string name="rt_apply_btn">Apply</string>
 
-    <string name="rt_download_desc">Download blocklists (around 60MB) to use this feature.</string>
+    <string name="rt_download_desc">Download blocklists (~60 MB) to use this feature.</string>
```

Comment:
```suggestion
    <string name="rt_download_desc">Download blocklists (around 60 MB) to use this feature.</string>
```
Since the other similar string is "Up to 350 MB"

Reply:
+1